release_docs/RELEASE.txt

```diff
@@ -572,6 +572,15 @@ Bug Fixes since HDF5-2.0.0 release
 
     Configuration
     -------------
+    - When using a system installed zlib library, the shared library is expected to
+      be found in the system library path.
+
+      Setting the HDF5_MODULE_MODE_ZLIB option to OFF will force find_package to
+      use config mode first, an installed zlib, or an alternate installed zlib
```

Comment:
Perhaps, "use config mode first, an installed zlib" should be "use config mode first.  An installed zlib"?